File: leetcode/binary_search/1060_Missing_Element_in_Sorted_Array/codes/brute.py
from typing import List

class Solution:
    def missingElement(self, nums: List[int], k: int) -> int:
        for i in range(1, len(nums)):
            gap = nums[i] - nums[i - 1] - 1
            if k <= gap:
                return nums[i - 1] + k
            k -= gap
        return nums[-1] + k

File: leetcode/binary_search/1060_Missing_Element_in_Sorted_Array/codes/brute.c
int missingElement(int* nums, int numsSize, int k) {
    for (int i = 1; i < numsSize; i++) {
        int gap = nums[i] - nums[i - 1] - 1;
        if (k <= gap) {
            return nums[i - 1] + k;
        }
        k -= gap;
    }
    return nums[numsSize - 1] + k;
}
